src/YesSql.Core/Configuration.cs

```diff
@@ -23,18 +23,31 @@ public Configuration()
             QueryGatingEnabled = true;
             Logger = NullLogger.Instance;
             ConcurrentTypes = new HashSet<Type>();
+            NamingPolicy = NamingPolicy.PascalCase;
```

Comment:
The default naming policy should be a NullNamingPolicy one. And keep whatever case is set by the user.

Reply:
> The default naming policy should be a NullNamingPolicy one

Ya, but if you look closely we're using PacalCase by default

Reply:
> User code should not have to deal with it, it should be part of the schema builder logic, the same way it will add the table prefix by itself.

I already did that for non columns, but I stuck in `CreateTable` because it passes the columns in the action delegete